src/config/entities/configuration.ts

```diff
@@ -394,4 +394,8 @@ export default () => ({
       },
     },
   },
+  jwt: {
+    issuer: process.env.JWT_ISSUER,
+    secret: process.env.JWT_SECRET,
+  },
```

Comment:
Nit: as before, let's move this to be alphabetical (the same for the test configuration).

Reply:
Done
The idea was to keep it so that we can decide whether we want to remove all the localized configuration values or not. But as you mentioned, let's delete it for now as we can always revert the changes.